src/Networking/v2/Extensions/Layer3/Api.php

```diff
@@ -26,6 +26,7 @@ public function postFloatingIps(): array
             'jsonKey' => 'floatingip',
             'params'  => [
                 'tenantId'          => $this->params->tenantIdJson(),
+                'description'       => $this->params->descriptionJson(),
```

Comment:
Description should be not required
```suggestion
                'description'       => $this->notRequired($this->params->descriptionJson()),
```